jobs/JGDAS_GLOBAL_OCEAN_ANALYSIS_PREP

```diff
@@ -2,36 +2,13 @@
 export STRICT="NO"
 source "${HOMEgfs}/ush/preamble.sh"
 
-#############################
-# Source relevant config files
-#############################
-export EXPDIR=${EXPDIR:-${HOMEgfs}/parm/config}
-configs="base ocnanal ocnanalprep"
-config_path=${EXPDIR:-${PACKAGEROOT}/gfs.${gfs_ver}/parm/config}
-for config in ${configs}; do
-    . "${config_path}"/config."${config}"
-    status=$?
-    [[ ${status} -ne 0 ]] && exit "${status}"
-done
-
-
-##########################################
-# Source machine runtime environment
-##########################################
-. "${HOMEgfs}"/env/"${machine}".env ocnanalprep
-status=$?
-[[ ${status} -ne 0 ]] && exit "${status}"
-
-
 ##############################################
-# Obtain unique process id (pid) and make temp directory
+# make temp directory
 ##############################################
-export pid=${pid:-$$}
-export outid=${outid:-"LL${job}"}
-
-export DATA=${DATA:-${DATAROOT}/ocnanal_${CDATE}}
-mkdir -p "${DATA}"
-cd "${DATA}" || (echo "${DATA} does not exist. ABORT!"; exit 1)
+export DATA=${DATA:-${DATAROOT}/ocnanal_${cyc}}
```

Comment:
@guillaumevernieres @CoryMartin-NOAA 
Take note here in the use of `${cyc}` instead of `${CDATE}`.

At this stage, we only have `$cyc` in ops coming from ecFlow.
In our development we provide `$PDY` and `$CDATE` via the Rocoto XML.

Please take special note of this.

Reply:
`setpdy.sh` sets `PDY` based on a file in the ops space.
If `PDY` is previously set (as is the case in development), `setpdy.sh` uses that info (with the `cycle`).

Reply:
Yeah so I guess my point is that we can assume we have `${PDY}${cyc}` at this point, right? Not that it really matters for this `${DATA}` directory.

Reply:
In `ops` `PDY` is set by the `setpdy.sh` script.
`setpdy.sh` needs to be executed while in the `DATA` directory as it creates a (pesky) file `ncepdate`.

As you can see, in this (and other) j-job, `DATA` directory name contains `PDY` which is undefined in `ops`.